### PR TITLE
[Movement v1] Path Planning z level check

### PR DESCRIPTION
**Pull request by maltewae**
Adds a check if target and start coordinates are approximately on the same z plane.

### CODE REVIEW

File: LabExT/Movement/PathPlanning.py [hunk 1]
```diff
@@ -247,11 +247,17 @@ def __init__(
...
-            y=target_coordinate.y,
-            z=self.start_coordinate.z)
+        # field target
+        self.target_coordinate = target_coordinate
+
+        if not np.isclose(
+                self.start_coordinate.z,
+                self.target_coordinate.z,
+                rtol=1.e-5,
+                atol=1.e-8,):
```
Comment (maltewae) [inline LabExT/Movement/PathPlanning.py:257]: I took the default values here. Are they ok, @marcoep ?

Comment (maltewae) [reply LabExT/Movement/PathPlanning.py]: Don't wonder, I just removed the comma after `atol=1.e-8,`

Comment (marcoep) [reply LabExT/Movement/PathPlanning.py]: I would set it such that it is okay within a +-10nm range. So keep `rtol` the same, but change `atol` to something like 10e-2 (assuming we are in units of microns).

Comment (maltewae) [reply LabExT/Movement/PathPlanning.py]: Changed it to 10e-2. Yes all units are in um.

Comment (marcoep) [reply LabExT/Movement/PathPlanning.py]: ahh, sorry, meant 10e-3 (i.e. 10nm)

Comment (maltewae) [reply LabExT/Movement/PathPlanning.py]: okay its 10e-3 now 😄

**Pull request merged**